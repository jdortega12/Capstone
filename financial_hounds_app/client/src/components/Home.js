import React from "react";
import "../styles/Home.css";

const Home = () => {
    return (
      <>
      <div className="home">
        <div className="headerContainer">
          <h1 className="header"> Loyola University Maryland </h1>
          <div>
          <p className="paragraph"> Greyhound Financial Planning</p>
          </div>
        </div>
      </div>
      </>
    );
  }
  
export default Home;