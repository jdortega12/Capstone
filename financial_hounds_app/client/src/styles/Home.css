.home {
  width: 100%;
  height: 100vh;
  display: flex;
  justify-content: center;
  align-items: center;
  flex-direction: column;
  background-color: #0d4b34;
  font-family: "Gill Sans", "Gill Sans MT", Calibri, "Trebuchet MS", sans-serif;
  
}

.headerContainer {
  height: auto;
  width: auto;
  visibility: visible;
  margin-left: 50px;
}

.header {
  margin-bottom: 50px;
  justify-content: center;
  align-items: center;
  font-size: 90px;
  height: 60px;
  font-weight: 50;
  color: white;
}

.paragraph {
  margin-left: 200px;
  justify-content: center;
  align-items: center;
  font-size: 40px;
  font-weight: lighter;
  color: white;
}

.link {
  margin-left: 400px;
  color: black;
  border: none;
  border-radius: 5px;
  cursor: pointer;
  justify-content: center;
  background-color: #282c34;
  color: white;
  text-decoration: none;
  font-size: 40px;
}

.headerContainer button:hover {
  background-color: #07090a;
  cursor: pointer;
  transition: 0.3s ease-in;
}

@media only screen and (max-width: 650px) {
  .home {
    justify-content: center;
    align-items: center;
  }

  .headerContainer {
    margin-left: 0px;
    border-radius: 10px;
    padding: 10px;
    width: 80%;
    display: flex;
    justify-content: center;
    align-items: center;
    flex-direction: column;
    background-color: #0d4b34;
    color: white;
  }

  .headerContainer h1 {
    font-size: 40px;
    height: 30px;
    color: white;
  }

  .headerContainer p {
    font-size: 30px;
    color: white;
  }

  .headerContainer button {
    background-color: white;
    color: #282c34;
    margin-bottom: 30px;
  }

  .headerContainer button:hover {
    background-color: rgb(225, 225, 225);
    color: #121619;
  }
}
