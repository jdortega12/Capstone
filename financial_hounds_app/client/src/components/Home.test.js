import React from 'react';
import Home from '../components/Home';
import renderer from 'react-test-renderer';

test("Home Page Snapshot Test", ()=> {
    const component = renderer.create(
        <Home />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});