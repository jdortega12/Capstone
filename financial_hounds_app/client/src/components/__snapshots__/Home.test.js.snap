// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page Snapshot Test 1`] = `
<div
  className="home"
>
  <div
    className="headerContainer"
  >
    <h1
      className="header"
    >
       Loyola University Maryland 
    </h1>
    <div>
      <p
        className="paragraph"
      >
         Greyhound Financial Planning
      </p>
    </div>
  </div>
</div>
`;
